web/src/app/services/auth.service.ts

```diff
@@ -53,63 +53,50 @@ export class AuthService {
       );
   }
 
-  formatUserResponse(user, provider): User {
+  formatUserResponse(response, provider): User {
     let normalisedUser = {};
+
+    if (typeof (response) === 'undefined') {
```

Comment:
Would `if (!response)` not work?

Reply:
I tested by passing an empty object as argument . The above condition which you mentioned was not working.

Reply:
ok